xml/Windows.UI.Xaml.Controls.Primitives/GeneratorPosition.xml

```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Describes the position of an item that is managed by Windows.UI.Xaml.Controls.ItemContainerGenerator.</summary>
```

Comment:
ItemContainerGenerator could be a cref, but I think the fully qualified type name is incorrect here -- I think that it's System.Windows.Controls.ItemContainerGenerator.

Reply:
Yeah, I wasn't sure here. I thought the same.

Reply:
Ok, I think this might actually be correct and link to the UWP type:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.itemcontainergenerator

I'll add the cref, since the xref link should work for UWP types too.